ftrip.io.catalog-service/ftrip.io.catalog-service/Startup.cs

```diff
@@ -38,6 +38,9 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
+            services.AddHttpClient("booking", client =>
+                client.BaseAddress = new Uri(Environment.GetEnvironmentVariable("BOOKING_SERVICE_URL"))
```

Comment:
Nice, you are using best practices. We should figure out how to add Correlation Id and Jwt Token when sending HTTP requests.

Reply:
We can add Jwt like this:
client.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue("...");

Reply:
That is true. Only question is how to get instances of contexts but we have services there so we can also get service provider for sure.